src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -12,6 +12,7 @@
 
 #include "dnn_types.hpp"
 #include "gna_lib_ver_selector.hpp"
+#include "gna_plugin_config.hpp"
```

Comment:
Why does this file require gna_plugin_config.hpp? I think gna_target.hpp would be enough

Reply:
Probably a leftover from some earlier version.